src/wbase/UserQueryInfo.cc

```diff
@@ -0,0 +1,91 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "wbase/UserQueryInfo.h"
+
+// System headers
+
+// Qserv headers
```

Comment:
This comment w/o any headers included after that is useless.

Reply:
But there is a qserv header.

Reply:
I don't see any `#include` statements after that comment.

Reply:
#include "util/Bug.h"

Reply:
I'm confused since I'm not seeing that include. If I open the file viewer to see the complete file then all I see is:
```
// Class header
#include "wbase/UserQueryInfo.h"

// System headers

// Qserv headers

// LSST headers
#include "lsst/log/Log.h"
```
Are you sure you have committed & pushed everything from your local GIt repo to this PR in GitHub?

Reply:
Line 25, on this page `// Qserv headers`
Line 26, on this page, directly below the comment I am typing now, `#include "util/Bug.h"`